microsoft/testsuites/network/sriov.py

```diff
@@ -693,12 +697,17 @@ def verify_sriov_interrupts_change(self, environment: Environment) -> None:
         server_iperf3.run_as_server_async()
         client_interrupt_inspector = client_node.tools[InterruptInspector]
         for _, client_nic_info in vm_nics[client_node.name].items():
+            if (
```

Comment:
Because the method is for sriov (PCI) only. Can it just run with `is_pci_module_enabled == true`? So it doesn't need a special logic here.

Reply:
updated